scripts/tag-version.sh

```diff
@@ -0,0 +1,4 @@
+CURR_VERSION=$(grep -o 'version=\"[0-9.]*\"' setup.py | sed "s/version=\"\([0-9.]*\)\"/\1/")
```

Comment:
Please include a shebang. `#!/bin/sh` should be fine. Same for other scripts. If bash is needed then `#!/usr/bin/env bash` should likewise be OK.